tests/test_plotting.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import pandas as pd
+import numpy as np
+from altair import Chart
+import os
+import sys
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'src', 'linreg_ally'))
```

Comment:
```suggestion
from linreg_ally.plotting import qq_and_residuals_plot
```
We do not need this kind of OS path manipulation if we use `poetry install`. All of it can be simplified if you follow the suggestion and use poetry to install the package to be editable.